crates/symbolicator/src/services/symbolication/process_minidump.rs

```diff
@@ -743,9 +709,18 @@ impl SymbolicationActor {
         let mut result = loop {
             iterations += 1;
 
-            let mut result = self
-                .stackwalk_minidump(minidump_path, cfi_caches, modules.is_none())
-                .await?;
+            let mut result = {
+                let return_modules = modules.is_none();
+                let cfi_caches = cfi_caches.for_processing();
+                let minidump_path = minidump_path.to_path_buf();
+
+                let future = tokio::time::timeout(
+                    Duration::from_secs(60),
+                    stackwalk(cfi_caches, minidump_path, return_modules)
+                        .bind_hub(sentry::Hub::current()),
+                );
+                self.cpu_pool.spawn(future).await???
```

Comment:
instead of spawning the timeout-wrapped future, we should timeout the spawned future ;-)

Hope that makes sense.

Also, I love the tripple-? :-D

Reply:
Is it correct now?

Reply:
yes, looks better!